import/Entities.xml

```diff
@@ -1,4 +1,4 @@
dDate="2020-08-19T17:57:03.341-04:00" name="DataStreamTemplate" projectName="FlexyConnector" published="false" tags="" thingTemplate="Stream" valueStream=""><Owner name="Administrator" type="User"/><avatar/><DesignTimePermissions><Create/><Read/><Update/><Delete/><Metadata/></DesignTimePermissions><RunTimePermissions/><VisibilityPermissions><Visibility/></VisibilityPermissions><ConfigurationTableDefinitions/><ConfigurationTables><ConfigurationTable description="Data Thing Configuration" isMultiRow="false" name="DataThingSettings" ordinal="0"><DataShape><FieldDefinitions><FieldDefinition baseType="STRING" description="Persistence Provider Name" name="persistenceProvider" ordinal="0"/></FieldDefinitions></DataShape><Rows><Row><persistenceProvider><![CDATA[ThingworxPersistenceProvider]]></persistenceProvider></Row></Rows></ConfigurationTable><ConfigurationTable description="Data Shape Configuration" isMultiRow="false" name="Settings" ordinal="0"><DataShape><FieldDefinitions><FieldDefinition aspect.friendlyName="Data Shape" baseType="DATASHAPENAME" description="Data shape" name="dataShape" ordinal="0"/></FieldDefinitions></DataShape><Rows><Row><accumulatedDataShape><json><![CDATA[{"fieldDefinitions":{"Value":{"name":"Value","aspects":{"isPrimaryKey":false},"description":"","baseType":"NUMBER","ordinal":1},"Name":{"name":"Name","aspects":{"isPrimaryKey":false},"description":"","baseType":"STRING","ordinal":0}}}]]></json></accumulatedDataShape><dataShape><![CDATA[LoggedInformation]]></dataShape></Row></Rows></ConfigurationTable><ConfigurationTable description="Configurable options to tune Stream performance" isMultiRow="false" name="PersistenceProviderCustomSettings" ordinal="2"><DataShape><FieldDefinitions><FieldDefinition aspect.friendlyName="Persistence Provider Custom Settings Table" baseType="INFOTABLE" description="Persistence Provider Custom Config Table" name="customConfigTable" ordinal="0"/></FieldDefinitions></DataShape><Rows><Row><customConfigTable><infoTable><DataShape><FieldDefinitions/></DataShape><Rows/></infoTable></customConfigTable></Row></Rows></ConfigurationTable></ConfigurationTables><ThingShape><PropertyDefinitions/><ServiceDefinitions><ServiceDefinition aspect.isAsync="false" category="" description="" isAllowOverride="false" isLocalOnly="false" isOpen="false" isPrivate="false" name="GetMostRecentDataEntry"><ResultType aspect.dataShape="LoggedInformation" aspect.isStreamEntry="true" baseType="INFOTABLE" description="" name="result" ordinal="0"/><ParameterDefinitions/></ServiceDefinition></ServiceDefinitions><EventDefinitions/><ServiceMappings/><ServiceImplementations><ServiceImplementation description="" handlerName="Script" name="GetMostRecentDataEntry"><ConfigurationTables><ConfigurationTable description="" isMultiRow="false" name="Script" ordinal="0"><DataShape><FieldDefinitions><FieldDefinition baseType="STRING" description="code" name="code" ordinal="0"/></FieldDefinitions></DataShape><Rows><Row><code><![CDATA[var params = {
```

Comment:
'Switch to export file from Thingworx 9' isn't very clear. Was the export file upgraded to Thingworx 9?

Reply:
Ill change to commit message to "export file upgraded to Thingworx 9"

Reply:
I don't know if you saw, but this pull request has merge conflicts. You'll need to run `git rebase` to get the latest changes from the `main` branch into your branch. Here is a document that describes a bit better: https://docs.openstack.org/doc-contrib-guide/additional-git-workflow/rebase.html (that document refers to `master`, but this repository uses `main` as the primary branch)

Reply:
Fixed, I think.